superset/views/core.py

```diff
@@ -1085,8 +1085,14 @@ def save_dash(  # pylint: disable=no-self-use
         DashboardDAO.set_dash_metadata(dash, data)
         session.merge(dash)
         session.commit()
+
+        # get updated changed_on
+        dash = session.query(Dashboard).get(dashboard_id)
+        last_modified_time = dash.changed_on.replace(microsecond=0).timestamp()
```

Comment:
Nit: `.replace(microsecond=0)` is probably not needed.

Reply:
i want to use timestamp in the second level, to avoid comparing 2 float number.